prefect_kubernetes/flows.py

```diff
@@ -0,0 +1,61 @@
+"""A module to define flows interacting with Kubernetes resources."""
+
+from typing import Any, Dict
+
+from prefect import flow, task
+
+from prefect_kubernetes.jobs import KubernetesJob
+
+
+@task
+async def trigger(kubernetes_job):
+    """Task for triggering a Kubernetes job."""
+    return await kubernetes_job.trigger()
+
+
+@task
+async def wait_for_completion(kubernetes_job_run):
+    """Task for waiting for a Kubernetes job to complete."""
+    await kubernetes_job_run.wait_for_completion()
+
+
+@task
+async def fetch_result(kubernetes_job_run):
+    """Task for fetching the result of a Kubernetes job."""
+    return await kubernetes_job_run.fetch_result()
```

Comment:
So just to confirm, we can't just wrap task like `task(fetch_result)`? Not sure if we want to expose this as public tasks. Maybe if we can't wrap task like that, we can nest this under the function.

Reply:
we definitely can wrap them, I just thought the decorator format was more readable

Reply:
If we go with the decorator format, maybe these can be private methods? I don't think these tasks are too useful alone.